packages/react-chat/src/components/UserResponse/index.tsx

```diff
@@ -36,11 +39,14 @@ export interface UserResponseProps {
 const UserResponse: React.FC<UserResponseProps> = ({ message, timestamp, debug }) => {
   useAutoScroll();
 
+  const { config } = useContext(RuntimeStateAPIContext);
   return (
-    <Container>
+    <Container mode={config.render!.mode}>
       <Row>
         <Timestamp value={timestamp} />
-        <Message from="user">{message}</Message>
+        <Message from="user" mode={config.render!.mode}>
```

Comment:
instead of drilling `mode` all the way down to individual components. How do we feel about just putting the styling into a parent component where it determines what it's children classes will be?
I think we do this in a few places already

Reply:
it was a bit of a speedy solution to drill since we were short on time, there's a ticket for this on DX side https://linear.app/voiceflow/issue/DX-1025/refactor-chat-modes-into-stitches-themes

Reply:
yeah we're planning to refactor this to use a theme in the future 👍